runtime/nls/j9vm/j9vm_de.nls

```diff
@@ -2165,15 +2165,15 @@ J9NLS_VM_CRIU_RESTORE_RESET_VIRTUALTHREAD_FORKJOINPOOL_PARALLELISM_INVALID_PARAL
 # END NON-TRANSLATABLE
 
 # NullRestricted/value/implicit is not translatable
```

Comment:
Perhaps we should update the base file to remove `value` and `implicit` from the list of words not to be translated (they appear to be translated here, and that seems appropriate).

Reply:
Updated this one.